crates/bindings-csharp/Runtime/LogStopwatch.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Text;
+using SpacetimeDB.Internal;
+
+namespace SpacetimeDB;
+
+public sealed class LogStopwatch : IDisposable
+{
+    private readonly FFI.ConsoleTimerId StopwatchId;
+    private bool WasStopped;
+
+    public LogStopwatch(string name)
```

Comment:
Looks like we need to update the proposal, it still has `private` constructor + a separate `Start()` method.